src/java.base/share/classes/java/util/Objects.java

```diff
@@ -68,7 +68,9 @@ public static boolean equals( @Nullable Object a,  @Nullable Object b) {
     }
 
     // Define and annotate an overload that is unique to j2cl: b/201433789
-    public static boolean equals( @Nullable String a,  @Nullable String b);
+    public static boolean equals( @Nullable String a,  @Nullable String b) {
+        return equals(a, b);
```

Comment:
I think you answered your own question :)

```suggestion
        return equals((Object) a, b);
```

Reply:
Duh! Thanks for noticing.